wp_api/src/api_error.rs

```diff
@@ -217,6 +219,26 @@ pub enum WpErrorCode {
     InvalidFeaturedMedia,
     #[serde(rename = "rest_no_authenticated_app_password")]
     NoAuthenticatedAppPassword,
+    #[serde(rename = "rest_no_featured_media")]
+    NoFeaturedMedia,
+    #[serde(rename = "rest_upload_file_error")]
+    UploadFileError,
+    #[serde(rename = "rest_upload_file_too_big")]
+    UploadFileTooBig,
+    #[serde(rename = "rest_upload_hash_mismatch")]
+    UploadHashMismatch,
+    #[serde(rename = "rest_upload_invalid_disposition")]
+    UploadInvalidDisposition,
+    #[serde(rename = "rest_upload_limited_space")]
+    UploadLimitedSpace,
+    #[serde(rename = "rest_upload_no_content_disposition")]
+    UploadNoContentDisposition,
+    #[serde(rename = "rest_upload_no_content_type")]
+    UploadNoContentType,
```

Comment:
I tried to add tests for these, as well as `UploadInvalidDisposition`, but had no luck.